cernopendata_client/cli.py

```diff
@@ -40,20 +30,26 @@ def cernopendata_client():
 @cernopendata_client.command()
 @click.option("--recid", type=click.INT, help="Record ID")
 @click.option("--doi", help="Digital Object Identifier.")
-@click.option("--title", help="Record title")
+@click.option("--title", help="Record Title")
```

Comment:
`Record title` is OK, we mostly use British English where titles are not usually that capitalised as in US English...

That said, we should amend help texts later, to provide nicer examples for people reading help pages like https://cernopendata-client.readthedocs.io/en/latest/cliapi.html#cernopendata-client-get-file-locations  I can expand that later.